# config/aruco_loc.yaml
# Parameters for ArUco localization
# Auto-generated - Edit with caution

---
/**:
  ros__parameters:
    general:
      marker_count: 3
      marker_size: 2
      maximum_distance: 2.0
      detection_rate: 0.5
      camera_name: zed
      world_frame_id: map
      refine_detection: false
    debug:
      level: 1
    marker_000:
      aruco_id: 0
      position:
        - 0
        - 0
        - 0
      orientation:
        - 0
        - 0
        - 0
    marker_001:
      aruco_id: 1
      position:
        - 0
        - 0
        - 0
      orientation:
        - 0
        - 0
        - 0
    marker_002:
      aruco_id: 2
      position:
        - 0
        - 0
        - 0
      orientation:
        - 0
        - 0
        - 0